screens/AppLayout.tsx

```diff
@@ -31,6 +28,13 @@ export const AppLayout: React.FC = () => {
         ))}
         {controller.isAuthorized &&
           authRoutes.map((route) => <Screen key={route.name} {...route} />)}
+        <MessageOverlay
```

Comment:
I'm moving this to the `WelcomeScreen.tsx` so that the popup can be shown in the app-launch screen. Please let me know if I should put this component in some other screen @vijay151096 @tilak-puli.